models/icepack/readme.rst

```diff
@@ -0,0 +1,81 @@
+
+The steps to run this example are as follows:
+
+1.  | ``cd /DART/models/icepack/work``
+    | Navigate to the work directory for Icepack
+
+2.  | ``./quickbuild.sh`` (or ``quickbuild.sh nompi`` if you are not building with mpi)
+    | Builds all DART executables 
+
+3.  | ``cp -r /glade/work/masmith/test_cases/icepack_test ..``
+    | Copy the test directory from the directory stated above to your Icepack directory
+
+4.  | ``cd ../icepack_test``
+    | Navigate to the test directory
+
+5.  | In ``work/input.nml``, set ``perturb_from_single_instance = .true.`` in the
+      ``&filter_nml``
+    | This setting causes filter to perturb a single restart file to generate an
+      ensemble
+
+6.  | ``./filter``
+    | Runs the assimilation program, resulting in four main output files:
+    |    ``analysis_mean.nc`` and ``analysis_sd`` - the mean and standard deviation of the state of all ensemble members after the assimilation
+    |    ``obs_seq.final`` - the ensemble members' estimate of the observations.
+    |    ``dart_log.out`` - detailed log file for the execution of filter
+
+Namelist
+--------
+
+.. code-block:: fortran
+
+   &model_nml
+      model_perturbation_amplitude = 2e-05
+      debug = 1
+      model_state_variables = 'aicen', 'QTY_SEAICE_CONCENTR', 'UPDATE', 'vicen',
+                            'QTY_SEAICE_VOLUME', 'UPDATE', 'vsnon', 'QTY_SEAICE_SNOWVOLUME',
+                            'UPDATE'
+      grid_oi = 3
+   /
+
+Description of each namelist entry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++------------------------------+---------------+---------------------------------+
+| Item                         | Type          | Description                     |
++==============================+===============+=================================+
+| model_perturbation_amplitude | real(r8)      | Perturbation amplitude          |
++------------------------------+---------------+---------------------------------+
+| debug                        | integer       | When set to 0, debug statements |
+|                              |               | are not printed. Higher numbers |
+|                              |               | mean more debug reporting.      |
++------------------------------+---------------+---------------------------------+
+| model_state_variables        | character(*)  | List of model state variables   |
++------------------------------+---------------+---------------------------------+
+| grid_oi                      | integer       | Specifies a constant to be used |
+|                              |               | as the value for the first      |
+|                              |               | dimension in calls to           |
+|                              |               | get_dart_vector_index           |
```

Comment:
what is grid_oi?  Why does it need to be a constant? 
What shape are the netcdf variables?